### PR TITLE
fix: add<Abdulwahid7027>

### LINKED ISSUE #1628: [PROFILE ADDITION] <Shaik Abdul Wahid>
### Profile screenshot

github com_Abdulwahid7027

### PR DESCRIPTION
**Pull request by Abdulwahid7027**
resolves #1628 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Abdulwahid7027.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/110891341/211197326-db82ec3c-ad43-4e98-ab85-6332152f9dd8.png",
```
Comment (Panquesito7) [inline profiles/Abdulwahid7027.json:2]: I recommend more of an image like this. What do you think?

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/211386495-2f54c6c3-dc32-4f15-ac29-cc541f8b6d89.png",
```

Comment (Abdulwahid7027) [reply profiles/Abdulwahid7027.json]: It's good.

**Review by krshkun — APPROVED** [review]
Looks good to me

**Review by Abdulwahid7027 — COMMENTED** [review]
Thanks

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Arindam200 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**